Origin: Debian
Forwarded: not yet
Last-Update: 2022-10-12

Index: gdb/gdb/printcmd.c
===================================================================
--- gdb.orig/gdb/printcmd.c
+++ gdb/gdb/printcmd.c
@@ -1877,7 +1877,7 @@ x_command (const char *exp, int from_tty
     {
       /* '*((int *(*) (void)) __errno_location) ()' is incompatible with
          function descriptors.  */
-      if (target_has_execution && strcmp (exp, "errno") == 0)
+      if (strcmp (exp, "errno") == 0)
           exp = "*(*(int *(*)(void)) __errno_location) ()";
       expression_up expr = parse_expression (exp);
       /* Cause expression not to be there any more if this command is
